build.sh

```diff
@@ -174,3 +174,7 @@ if [ "$format_error_files" != "" ]; then
   exit 42
 fi
 
+if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
```

Comment:
The logic is a bit backward here (why not `== "true"`)? I am not an expert in shell programming, but from configure scripts I seem to remember that test are often written as `x$foo != xabc`. I believe this is to avoid problems in case `$foo` is empty.